extensions/CreateBox/CreateBox.php

```diff
@@ -130,10 +130,15 @@ function acMakeBox( $input, $argv, $parser ) {
 
 function acRedirect( $title, $action ) {
 	global $wgRequest, $wgOut;
-	$query = "action={$action}"
-		. '&preload=' . $wgRequest->getVal( 'preload' )
-		. '&editintro=' . $wgRequest->getVal( 'editintro' )
-		. '&section=' . $wgRequest->getVal( 'section' );
+
+	$query = http_build_query([
+		'action' => $action,
+		'preload' => $wgRequest->getVal( 'preload' ),
+		'editintro' => $wgRequest->getVal( 'editintro' ),
+		'section' => $wgRequest->getVal( 'section' ),
+		'flow' => FlowTrackingHooks::CREATE_PAGE_CREATE_BOX
```

Comment:
This might fail in case that flow tracking extension is disabled. Can we do something about that?